p2p/libp2p/discovery/kadDhtDiscoverer.go

```diff
@@ -127,7 +136,7 @@ func (kdd *KadDhtDiscoverer) connectToInitialAndBootstrap() {
 			i := 1
 			for {
 				if kdd.initConns {
-					err := kdd.kadDHT.Bootstrap(ctx)
+					err := kdd.kadDHT.BootstrapOnce(ctx, cfg)
```

Comment:
BootstrapOnce vs Boostrap. Do they do the same things?

Reply:
yes, that was the name change between kad-dht implementation versions. https://github.com/libp2p/go-libp2p-kad-dht/releases/tag/v0.3.0

Reply:
ok